pkg/deploy_test.go

```diff
@@ -18,16 +20,23 @@ func TestGetDeployPlan(t *testing.T) {
 		t.FailNow()
 	}
 
-	wd, _ := os.Getwd()
-	fp := filepath.Join(wd, "..", "..", "_deploy")
+	var fp string
+	envdir := os.Getenv(constants.ENV_SKUID_DEFAULT_FOLDER)
+	if envdir != "" {
+		fp = envdir
+	} else {
+		wd, _ := os.Getwd()
+		fp = filepath.Join(wd, "..", "..", "_deploy")
+	}
+	fmt.Printf("using directory %s\n", fp)
```

Comment:
Did you want to leave this print statement in?

Reply:
lol no, ty.